cluster-autoscaler/utils/kubernetes/listers.go

```diff
@@ -193,18 +181,12 @@ func ScheduledPods(allPods []*apiv1.Pod) []*apiv1.Pod {
 	return scheduledPods
 }
 
-// SchedulerUnprocessedPods is a helper method that returns all pods which are not yet processed by the specified ignored schedulers
-func SchedulerUnprocessedPods(allPods []*apiv1.Pod, ignoredSchedulers map[string]bool) []*apiv1.Pod {
+// SchedulerUnprocessedPods is a helper method that returns all pods which are not yet processed by the specified bypassed schedulers
+func SchedulerUnprocessedPods(allPods []*apiv1.Pod, bypassedSchedulers map[string]bool) []*apiv1.Pod {
 	var unprocessedPods []*apiv1.Pod
 
-	isDefaultSchedulerIgnored := getIsDefaultSchedulerIgnored(ignoredSchedulers)
-
 	for _, pod := range allPods {
-		// Don't add a pod with a scheduler that isn't specified by the user
-		if !isDefaultSchedulerIgnored && pod.Spec.SchedulerName == "" {
-			continue
-		}
-		if isIgnored, found := ignoredSchedulers[pod.Spec.SchedulerName]; !found || !isIgnored {
+		if isIgnored, found := bypassedSchedulers[pod.Spec.SchedulerName]; !found || !isIgnored {
```

Comment:
Checking for presence of key in map is redundant, `isIgnored` always implies `found`. Besides, we're only storing `true` values there.

Also, nit: rename `isIgnored` to `canBypass`?

Reply:
Yes, updated now